Whiskey/Whiskey.psd1

```diff
@@ -146,37 +146,11 @@
 * You can now specify a custom version of NUnit 3 that the `NUnit3` task should use by setting the `Version` property to the version you want to use.
 * The `NUnit3` task upgraded to use NUnit 3.8.1 (from 3.7.0).
 
-Added support for `PackageName` and `DeployTo` properties to `Publish-WhiskeyBuildMasterPackage` task function, e.g.
-
-    PublishTasks:
-    - PublishBuildMasterPackage:
-        ApplicationName: TestApplication
-        Uri: https://buildmaster.example.com
-        ApiKeyID: buildmaster.example.com
-		PackageName: TestPackage
-        DeployTo:
-        - BranchName:
-          - develop
-		  - feature*
-	      ReleaseName: TestRelease
-          StartAtStage: Test
-        - BranchName: prod
-          ReleaseName: ProdRelease
-          SkipDeploy: true
-
-`PackageName` (optional): defines the name of the package that will be created in BuildMaster. In this example, the `PackageName` will be `TestPackage` instead of the default convention of `MajorVersion.MinorVersion.PatchVersion`  
-
-`DeployTo` (mandatory) defines a map of SCM branch to corresponding BuildMaster releases where packages should be created and deployed. `DeployTo` contains the following properties:
-- `BranchName` (mandatory) defines the SCM branch to be mapped to the release. Wildcards are allowed.
-- `ReleaseName` (mandatory) defines the release in BuildMaster where packages should be created and deployed.
-- `StartAtStage` (optional) defines the stage of the release pipeline where the package should start its deployment. By default, the package will be released to the first stage of the pipeline.
-- `SkipDeploy` (optional) defines that the release package should be created, but not automatically deployed. By default, the package deployment will be started.
-
-When building on `develop`, `feature/NewFunction`, or `feature68` branches, a package will be created on the `TestRelease` release of the `TestApplication` application. The package will be deployed to the `Test` stage of the `TestRelease` release's pipeline.
-
-When building on `prod` branch, a package will be created on the `ProdRelease` release of the `TestApplication` application. The package will be created, but will not be deployed.
-
-When building on `unmapped` branch, the task will fail with an error stating that the current branch must be mapped to a `ReleaseName`. No package will be created or deployed.
+Added support for `PackageName`, `StartAtStage`, and `SkipDeploy` properties to `Publish-WhiskeyBuildMasterPackage` task function.

```

Comment:
Change to:

```
* PublishBuildMasterPackage task now supports:
  * customizing package names with the `PackageName` property. The default value will continue to be the major, minor, and patch portions of the version number, e.g. `6.4.34`.
 * skipping/preventing a package from deploying with the `SkipDeploy` property. Set it to `true` and a package will be created in BuildMaster, but not deployed anywhere.
* starting a deploy at a specific stage of a release's pipeline. Set the `StartAtStage` property to the name of the stage you want the deploy to start at. In order for a deploy to start at any stage in a pipeline, the "Enforce pipeline stage order for deployments" setting must be off. This can be set in the BuildMaster UI in the pipeline's Edit panel.